bls/bls.go

```diff
@@ -0,0 +1,87 @@
+package bls
+
+// #cgo LDFLAGS: -L../ -ldusk_bls12_381_sign -framework Security
+// #include "../libbls.h"
+import "C"
+import (
+	"errors"
+	"unsafe"
+)
+
+func GenerateKeys() ([]byte, []byte) {
+	keysBuf := make([]byte, 128)
+	keys_ptr := toPtr(keysBuf)
+	C.generate_keys(keys_ptr)
+	return keysBuf[:32], keysBuf[32:]
+}
+
+func Sign(sk, pk, msg []byte) ([]byte, error) {
+	sk_ptr := toPtr(sk)
+	pk_ptr := toPtr(pk)
+	msg_ptr := toPtr(msg)
+	sigBuf := make([]byte, 48)
```

Comment:
Not going to add a comment for every length, but please change all of those to `C.SIG_SIZE`, `C.PK_SIZE` AND `C.SK_SIZE`.